platform/mk2/hal/gps_skytraq/gps_device_skytraq.c

```diff
@@ -29,9 +29,10 @@
 #define MESSAGE_TYPE_BINARY		2
 #define GNSS_NAVIGATION_MODE		GNSS_NAVIGATION_MODE_AUTOMOBILE
 
-#define BAUD_RATE_COUNT 2
+#define BAUD_RATE_COUNT 3
 #define BAUD_RATES 	{			\
 		{921600, 8},			\
+		{115200, 5},            	\
```

Comment:
Unplug your RCT for a good 30 seconds then look at the log again. it should default at 9600 baud.